server/queries/updateSurvey.js

```diff
@@ -43,12 +44,27 @@ const updateSurvey = async (surveyId, changes) => {
         });
       }
       const questionsFromSurvey = changes.questions;
+      questionsFromSurvey.map(async (question) => {
+        // if they have a _id prop, they exist already in the collection
```

Comment:
remove comment and replace question._id with a variable: `surveyExistsInCollection  = question._id`